src/nvim/spellfile.c

```diff
@@ -5378,7 +5378,9 @@ spell_add_word (
                    len, word, NameBuff);
             }
           }
-          fseek(fd, fpos_next, SEEK_SET);
+          if (fseek(fd, fpos_next, SEEK_SET) <= 0) {
+            ELOG("Error seeking to next position");
```

Comment:
If fseek() fails we should `break`, just like the case where `mch_fopen` fails a few lines above.